### PR TITLE
flight and hotel helper functions for API get requests 

### PR DESCRIPTION
**Pull request by kirkeaston**

### CODE REVIEW

File: server/lib/helpers/flightHelpers.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+// description link https://developers.amadeus.com/self-service/category/air/api-doc/flight-inspiration-search
+// answers the question "what are the cheapest flights from <current destination>"
+
+const Amadeus = require('amadeus');
+
+const amadeus = new Amadeus();
+
+const getFlightInspiration = (currentLocationCode, callback) => {
+  amadeus.shopping.flightDestinations.get({
+    origin: currentLocation,
```
Comment (Pix3lAssassin) [inline server/lib/helpers/flightHelpers.js]: Current location doesn't seem to be defined

File: server/lib/helpers/flightHelpers.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    departureDate,
+    numberOfAdults,
+  }).then((response) => {
+    console.log(response);
+    callback(null, response);
+  }).catch((response) => {
+    console.error('error getting flight in server', response);
+    callback(response, null);
+  });
+};
```
Comment (Pix3lAssassin) [inline server/lib/helpers/flightHelpers.js]: Could return the promise here

File: server/routes/flights.js [hunk 3]
```diff
@@ -1,3 +1,61 @@
...
+    originLocationCode: req.params.originLocationCode,
+    destinationLocationCode: req.params.destinationLocationCode,
+    departureDate: req.params.departureDate,
+    adults: req.params.adults,
+  }).then((response) => {
+    console.log(response);
+    res.status(200).send(response);
+  }).catch((err) => {
+    console.log('error in server getting flights', err);
+    res.status(404).send(err);
```
Comment (Pix3lAssassin) [inline server/routes/flights.js]: Should probably be status code 500 or similar

**Review by Pix3lAssassin — APPROVED** [review]

File: server/routes/flights.js (hunk 3 above)
Comment (kirkeaston) [reply server/routes/flights.js]: Oh, woops...

**Pull request merged**